### DISCUSSION TITLE
Support Alternative Payment Mechanism

### DISCUSSION THREAD

**Discussion by vHanda** [GitJournal/GitJournal#711] [Feature Requests]
This is required for F-Droid and a few users have contacted me regarding not using Google Play Store. I meant to have this done in October.

**Comment by finzzz** [comment]
Please reprioritize this! I want to buy but not through google.

**Comment by Narcolapser** [comment]
Looks like he has some sort of alternative mentioned here: https://www.reddit.com/r/GitJournal/comments/wvqkmc/payment_options/

Basically email him: [vhanda@gitjornal.io](mailto:vhanda@gitjornal.io)
